BGL/include/CGAL/boost/graph/helpers.h

```diff
@@ -537,6 +521,34 @@ make_quad(const P& p0, const P& p1, const P& p2, const P& p3, Graph& g)
 
 /** 
  * \ingroup PkgBGLHelperFct
+ * Creates an isolated quad with border edges in `g` having `p0`, `p1`, `p2`, and `p3` as points and adds it to the graph `g`.
+ * \returns the non-border halfedge which has the target vertex associated with `p0`.
+ **/ 
+template<typename Graph, typename P>
+typename boost::graph_traits<Graph>::halfedge_descriptor
+make_quad(const P& p0, const P& p1, const P& p2, const P& p3, Graph& g)
+{
+  typedef typename boost::graph_traits<Graph>              Traits;
+  typedef typename Traits::halfedge_descriptor             halfedge_descriptor;
+  typedef typename Traits::vertex_descriptor               vertex_descriptor;
+  typedef typename Traits::face_descriptor                 face_descriptor;
+  typedef typename boost::property_map<Graph,vertex_point_t>::type Point_property_map;
+  Point_property_map ppmap = get(CGAL::vertex_point, g);
+  vertex_descriptor v0, v1, v2, v3;
+  v0 = add_vertex(g);
+  v1 = add_vertex(g);
+  v2 = add_vertex(g);
+  v3 = add_vertex(g);
+
+  ppmap[v0] = p0;
+  ppmap[v1] = p1;
+  ppmap[v2] = p2;
+  ppmap[v3] = p3;
+  return make_quad(v0, v1, v2, v3);
```

Comment:
That does not compile. Please test your changes locally to avoid red lines:
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.10-Ic-80.shtml#BGL